From 784e46afd7530f3010726ba73cc1a81059eae8bf Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Mon, 5 Jun 2017 13:26:18 +0200
Subject: [PATCH] cmake: allow build without c++

By default cmake also searches for a c++ compiler:
https://cmake.org/cmake/help/v3.7/command/project.html
"By default C and CXX are enabled if no language options are given."

This patch restricts the compiler to C to fix a build error detected
by the buildroot autobuilders:
http://autobuild.buildroot.net/results/c04/c04ff29b83c020573add597c355c57a6d9c9761d/build-end.log

Patch sent upstream: https://github.com/hsleisink/hiawatha/pull/30

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4f28d76..e8051fc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.0)
-project(Hiawatha VERSION 10.6)
+project(Hiawatha VERSION 10.6 LANGUAGES C)
 
 # Compiler
 set(CMAKE_C_FLAGS "-O2 -Wall -Wextra ${CMAKE_C_FLAGS}")
-- 
2.11.0

